OutOfSchool/OutOfSchool.WebApi/Mapping/Extensions/MapExtension.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using AutoMapper;
+
+namespace OutOfSchool.WebApi.Mapping.Extensions
+{
+    public static class MapExtension
+    {
+
+        public static TDestination Map<TSource, TDestination>(this TSource source, Action<IMapperConfigurationExpression> configure)
+        {
+            var config = new MapperConfiguration(configure);
+            var mapper = config.CreateMapper();
+            var destination = mapper.Map<TDestination>(source);
+            return destination;
+        }
+
+        public static TDestination Map<TSource, TDestination>(this TSource source)
+        {
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<TSource, TDestination>().MaxDepth(1);
+            });
+            var mapper = config.CreateMapper();
+            var destination = mapper.Map<TDestination>(source);
+            return destination;
+        }
+
+
+        public static TDestination Map<TSource, TDestination>(this TSource source, TDestination destination)
```

Comment:
this and Map<TSource, TDestination>(this TSource source) methods
do we need both of them?